app/core/ingestion/pipeline.py

```diff
@@ -23,43 +24,133 @@
+        .subquery()
+    )
+
+
+def generate_pipeline_ingest_input_query(
+    db: Session,
+) -> Sequence[
+    Tuple[
+        Family,
+        FamilyDocument,
+        FamilyMetadata,
+        list[str],
+        Organisation,
+        Corpus,
+        PhysicalDocument,
+    ]
+]:
+    """Get a list of non-deleted docs and their associated meta & geos.
+
+    This function combines the results of three separate queries:
+    family and metadata, documents, and geographies by iterating over
+    the results of these queries, matching families with their documents
+    and geographies based on the family_import_id. The final result is a
+    list of tuples, containing the required information to construct a
+    DocumentParserInput object.
+
+    :param Session db: The db session to query against.
+    :return Sequence[Tuple[
+        Family,
+        FamilyDocument,
+        FamilyMetadata,
+        list[str],
+        Organisation,
+        Corpus,
+        PhysicalDocument
+    ]]: A list of tuples containing the information needed to populate
+        a DocumentParserInput object.
+    """
+
+    _LOGGER.info("Combining results of pipeline subqueries")
 
+    # Subquery to aggregate geographies
+    geography_subquery = fetch_geographies(db)
+
+    # Main query
     query = (
         db.query(
-            Family, FamilyDocument, FamilyMetadata, geo_subquery.c.value, Organisation, Corpus, PhysicalDocument  # type: ignore
+            Family,
+            FamilyDocument,
+            FamilyMetadata,
+            geography_subquery.c.geographies,  # type: ignore
+            Organisation,
+            Corpus,
+            PhysicalDocument,
         )
+        .select_from(FamilyDocument)
         .join(Family, Family.import_id == FamilyDocument.family_import_id)
-        .join(FamilyCorpus, FamilyCorpus.family_import_id == Family.import_id)
-        .join(Corpus, Corpus.import_id == FamilyCorpus.corpus_import_id)
         .join(FamilyMetadata, Family.import_id == FamilyMetadata.family_import_id)
+        .join(FamilyCorpus, Family.import_id == FamilyCorpus.family_import_id)
+        .join(Corpus, Corpus.import_id == FamilyCorpus.corpus_import_id)
         .join(Organisation, Organisation.id == Corpus.organisation_id)
         .join(
             PhysicalDocument, PhysicalDocument.id == FamilyDocument.physical_document_id
         )
+        .join(
+            geography_subquery,
+            geography_subquery.c.family_import_id == Family.import_id,  # type: ignore
+        )
         .filter(FamilyDocument.document_status != DocumentStatus.DELETED)
-        .filter(geo_subquery.c.family_import_id == Family.import_id)  # type: ignore
+        .filter(geography_subquery.c.family_import_id == Family.import_id)  # type: ignore
+        .options(
+            # Disable any default eager loading as this was causing multiplicity due to
+            # implicit joins in relationships on the selected models.
+            lazyload("*")
+        )
```

Comment:
We talked on slack about this, the fact that we are effectively having to turn off the ORM, makes me think we have finally outgrown the use of it for this query (And maybe we did a few changes ago!) ORMs are strong on small transactional queries, but are typically abandoned for more OLAP-y queries (similar to the approach with the all data download). I've suggested switching to sql for this pipeline query as I'm concerned the lazy loading option will trip us up at some point in the future as we make db_client changes. Katy mentioned she was already using a sql version for development so hoping its an easier switch, despite there will be some result parse changes required below